stdlib/_socket.pyi

```diff
@@ -5,6 +5,9 @@ from socket import error as error, gaierror as gaierror, herror as herror, timeo
 from typing import Any, SupportsIndex, overload
 from typing_extensions import TypeAlias
 
+if sys.version_info >= (3, 13):
+    from types import CapsuleType
```

Comment:
I think we backport this in `typing_extensions`; you should be able to get that on all versions

Reply:
ah, thanks!